src/components/loader-spinner/loader-spinner.config.ts

```diff
@@ -0,0 +1,52 @@
+type LoaderSpinnerSizes =
```

Comment:
suggestion (non-blocking): you can derive the types from the array here if you want

```
const LOADER_SPINNER_VARIANTS = [
  "action",
  "neutral",
  "inverse",
  "gradient-grey",
  "gradient-white",
] as const;

type LoaderSpinnerVariants = typeof LOADER_SPINNER_VARIANTS[number];

const LOADER_SPINNER_SIZES = [
  "extra-small",
  "small",
  "medium",
  "large",
  "extra-large",
] as const;

type LoaderSpinnerSizes = typeof LOADER_SPINNER_SIZES[number];